.gitignore

```diff
@@ -1,3 +1,6 @@
+dev_handler
+shm_api.c
```

Comment:
why is this ignored?

Reply:
dev_handler binary makes sense
shm_api.c I have no idea. Maybe ask runtime?

Reply:
oh shm_api.c is generated by cypthon when running make. Will add comment